swingset-demo/src/main/java/com/nqadmin/swingset/demo/TestBaseComponents.java

```diff
@@ -315,6 +317,7 @@ public void performRefreshOps() {
 			// SETUP NAVIGATOR QUERY
 				final String query = "SELECT * FROM swingset_base_test_data;";
 				cmbSSDBComboNav = new SSDBComboBox(connection, query, "swingset_base_test_pk", "swingset_base_test_pk");
+				cmbSSDBComboNav.setLogColumnName("combo_nav");
 
```

Comment:
Otherwise there's a `null` in the log entries.

Reply:
I like it. Maybe ** combo navigator ** so that it stands out from normal db column?

Reply:
OK, in SSSyncManager constructor I'll set one that stands out if it hasn't already been set. And it can be changed later.